### PR TITLE
fix(core): :bug: corelistitem validvalues are not in array

### PR DESCRIPTION
**Pull request by arkaprota**
## Description

the requirement to put the validvalues are in array

Ref: #328

### CODE REVIEW

File: package/components/dataDisplay/CoreListItem.js [hunk 1]
```diff
@@ -121,7 +121,7 @@ CoreListItem.validProps = [
       {
         default    : {},
         type       : "object",
-        validValues: "{ root?: elementType }"
+        validValues: ["{ root?: elementType }"]
```
Comment (techoneel) [inline package/components/dataDisplay/CoreListItem.js]: valid values are wrongly placed.

**Review by techoneel — CHANGES_REQUESTED** [review]
Please see the changes and give your reference to this.

**Comment by arkaprota** [comment]
please check if there is any changes required or not
@techoneel

File: package/components/dataDisplay/CoreListItem.js (hunk 1 above)
Comment (arkaprota) [reply package/components/dataDisplay/CoreListItem.js]: validvalues are removed please let me know if there is any changes needed or not
@techoneel

**Review by techoneel — APPROVED** [review]
good to go

**Pull request merged**